src/index.html

```diff
@@ -1,13 +1,393 @@
+        <div class="page__cap">
+          <h2 class="page__title title2">Новини</h2>
+
+          <button onclick="window.location.href='#news'" class="page__button-transparent page__button-top">
+            Усі новини
+
+            <span class="page__button-arrow"></span>
+          </button>
+        </div>
+
+        <div class="news__cards">
+          <div class="news__card">
+            <div class="news__image">
+              <div class="news__photos"></div>
+            </div>
+
+            <p class="news__pre-century">9 серпня 2019</p>
+
+            <h3 class="news__subtitle">Оголошення переможця</h3>
+
+            <p class="news__description">Друзі, сьогодні п'ятниця! А це означає, що час оголосити переможця розіграшу...</p>
+          </div>
+
+          <div class="news__card">
+            <div class="news__image">
+              <div class="news__cat"></div>
+            </div>
+
+            <p class="news__pre-century">9 серпня 2019</p>
+
+            <h3 class="news__subtitle">Міжнародний день котів</h3>
+
+            <p class="news__description">Музей з левами не може просто так взяти і пропустити Міжнародний день котів!</p>
+          </div>
+        </div>
+
+        <button onclick="window.location.href='#news'" class="page__button-transparent page__button-bottom">
+          Усі новини
+
+          <span class="page__button-arrow"></span>
+        </button>
+      </section>
+
+      <section id="subscribe" class="subscribe container">
+        <div class="page__cap subscribe__cap">
+          <h2 class="subscribe__title title2">Підпишіться на дайджест</h2>
+
+          <p class="subscribe__description">Першими дізнавайтесь про новини музею та розіграші, отримуйте запрошення на події та читайте статті від кураторів</p>
+        </div>
+
+        <form
+          method="post"
+          action="#form">
+          <input
+            autocomplete="email"
+            class="subscribe__input"
+            type="email"
+            name="email"
+            placeholder="e-mail"
+            required
+          >
+          <button class="page__button subscribe__button" type="submit">Підписатись</button>
+        </form>
+      </section>
+    </main>
+    <footer>
+      <div class="page__background">
+        <section class="footer container">
+          <section class="footer__content">
+            <div class="footer__contacts">
+              <h5 class="footer__title">Контакти</h5>
+
+              <ul class="footer__list-contacts">
+                <li class="footer__item-contacts">
+                  <a
+                  target="_blank"
+                  href="https://tinyurl.com/3brrvxy3"
+                  class="footer__link-contacts"
+                >Київ, <br> вул. М. Грушевського, 6</a>
+                </li>
+                <li class="footer__item-contacts">
+                  <a href="tel:044-278-74-54" class="footer__link-contacts">тел. 278-13-57, 278-74-54</a>
```

Comment:
Need to fix format phone in href
```suggestion
                  <a href="tel:+380442787454" class="footer__link-contacts">тел. 278-13-57, 278-74-54</a>
```